src/main/resources/site/i18n/phrases.properties

```diff
@@ -282,6 +282,10 @@ kpiErrorCalculationFailed = Beregning feilet
 kpiErrorUnknownError = Ukjent feil, prøv igjen
 readAboutCalculator = Les om kalkulatoren
 calculatorMonthAverageFrontpage = Alle
+calculatorResultScreenReader = Beløpet tilsvarer {0} kroner. {1} {2} prosent fra {3} til {4}. Indeks for {3} er {5}. Indeks for {4} er {6}
+calculatorValidateYear = Du kan kun skrive inn årstall fra {0} til
```

Comment:
Samme her, bruk denne for pif kalkulatoren så du kan slette pifValidateYear som er duplikat